eval/compile_op.go

```diff
@@ -140,10 +140,12 @@ func (op *pipelineOp) Invoke(fm *Frame) error {
 			if err != nil {
 				msg += ", errors = " + err.Error()
 			}
-			if fm.Editor != nil {
-				fm.Editor.Notify("%s", msg)
-			} else {
-				fm.ports[2].File.WriteString(msg + "\n")
+			if fm.Evaler.notifyBgJobSuccess != true || err != nil {
```

Comment:
Since we reverted the meaning of the flag, it should simply be

`if fm.Evaler.notifyBgJobSuccess || err != nil`

(note that even if you want `!= true`, you would always write `!flag` instead of `flag != true`)